### DISCUSSION TITLE
section и селект с where

### DISCUSSION THREAD

**Discussion by alexpash30** [totumonline/totum-ru-issues-and-discussions#565] [ОШБК!] [answered]
Добрый день.
Не работает section в селекте, если ссылаешься на селект с условием where. 
Или так и должно быть?

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Что имеется в виду под "ссылаешься". Если имеется в виду, что селект собран склеиванием например из двух или с добавлением элементов то да, точно не работает. Но если просто srlfs ... where то должно.
